src/components/navigation/footer/Footer.tsx

```diff
@@ -1,90 +1,156 @@
   navigationData,
-  companyInfo,
-  brandAssets,
   soMeData,
-  legalData,
+  /* legalData, */
+  companyInfo,
+  companyLocations,
 }: IFooter) => {
-  const currentYear = new Date().getFullYear();
+  const t = useTranslations("footer");
 
   return (
     <footer className={styles.footer}>
-      {brandAssets?.secondaryLogo && (
-        <div className={styles.logo}>
-          <SanityImage image={brandAssets.secondaryLogo} />
-        </div>
-      )}
-      <nav className={styles.nav}>
-        {renderLinks(navigationData)}
-        {soMeData && renderSoMe(navigationData, soMeData)}
-      </nav>
-      <ul className={styles.credits}>
-        <li key="credit-legal-key-1">
-          <Text className={styles.whiteColor}>
-            {`© ${currentYear} ${companyInfo.companyName}`}
-          </Text>
-        </li>
-        {legalData?.map((legal) => {
-          const link: ILink = {
-            _key: legal._id,
-            _type: legal._type,
-            linkTitle: legal.basicTitle,
-            linkType: LinkType.Internal,
-            internalLink: {
-              _ref: legal.slug.current,
-            },
-            language: legal.language,
-          };
-          return (
-            <li key={legal._id}>
-              <CustomLink link={link} type="footerLink" />
+      <FooterIllustration color={"#FFD02F"} />
+      <div className={styles.footerContent}>
+        <nav className={styles.nav}>
+          <div className={styles.flex_container_left}>
+            <div>
+              <TextTertiary>{t("text")}</TextTertiary>
+              {/*  TODO:: Get offices and map through. Should these link to something?  */}
+              <ul className={styles.offices}>
+                {companyLocations.map((location) => (
+                  <li
+                    key={location.companyLocationName}
+                    className={styles.dotSeparator}
+                  >
+                    {location.companyLocationName}
+                  </li>
+                ))}
+              </ul>
+            </div>
+            <div>
+              <FooterSection title={t("contact")}>
+                <Link
+                  href={`mailto:${companyInfo.companyEmail}`}
+                  className={styles.contactInfo}
+                >
+                  {companyInfo.companyEmail}
+                </Link>
+              </FooterSection>
+            </div>
+          </div>
+          <div className={styles.flex_container_right}>
+            {navigationData.main && (
+              <FooterSection title={t("mainMenu")}>
+                {renderPageLinks(navigationData)}
+              </FooterSection>
+            )}
+            {navigationData.footer && (
+              <FooterSection title={t("other")}>
+                {renderOtherLinks(navigationData)}
+              </FooterSection>
+            )}
+
+            {soMeData && (
+              <FooterSection title={t("socialMedia")}>
+                {renderSoMe(navigationData, soMeData)}
```

Comment:
Typically this should be a component to utilise react properly, but can be fixed later

Reply:
Yep, there are also a few things that can be done to make this more dynamic and utilize the connection to Sanity better, but I would rather get a version of this out now, and then we can make it even better later 👌

Regarding max-width I wasn't quite sure if it should have it or not, as some footers span the width of the page, but it won't be a problem to add it if we see that it's too big in relation to other content